signing-key.pub

```diff
@@ -0,0 +1,9 @@
+-----BEGIN PUBLIC KEY-----
+MIIBIjANBgkqhkiG9w0BAQEFAAOCAQ8AMIIBCgKCAQEAyGgvuy6CWSgJuhKY8oVz
+42udH1F2yIlaBoxAdQFuY2zxPSSpK9zv34B7m0JekuC5WCYfW0gS2Z8Ryu2RVdQh
+7DXvQb7qwzZT0H11K9Pw8hIHBvZPM+d61GWgWDc3k/rFwMmqd+kytVZy0mVxNdv4
+P2qvy6BNaiUI7yoB1ahR/6klfkPit0X7pkK9sTHwW+/WcYitTQKnEnRzA3q8EmA7
+rbU/sFEypzBA3C3qNJZyKSwy47kWXhC4xXUS2NXvew4FoVU6ybMoeDApwsx1AgTu
+CPPnPlCwuCIyUPezCP5XYczuHfaWeuwArlwdJFSUpMTc+SqO6REKgL9yvpqsO5Ia
+sQIDAQAB
+-----END PUBLIC KEY-----
```

Comment:
Really wondering if we should just host this somewhere instead?

Cc: @brent-phylum in case he's got a good idea of where we could store this so that people can download it.

Reply:
I don't like the idea of adding files to the top level, but I also don't think there is a better spot at the moment. I'd be cool to have it hosted somewhere on phylum.io but I think it's also worth having it in VCS as well, just for redundancy. So I'd say let's go ahead and commit this.

Reply:
We do have the [`assets`](https://github.com/phylum-dev/cli/tree/main/assets) folder, which I considered. Or we could put it in [`scripts`](https://github.com/phylum-dev/cli/tree/main/scripts) next to the script which uses it. I'm not sure I have a strong preference.

Reply:
`assets` feels a bit unrelated, given the context; `scripts` would be semantically a bit weird but it would also make sense for the file to be next to the script that's using it. I don't have a strong preference either, if I had to pick I'd probably go with `scripts`.